### PR TITLE
SorrTask_Replace_get_DagBuilder_name_from_string_pointer

### PR DESCRIPTION
**Pull request by samarth9008**
- Replace `get_DagBuilder_name_from_string_pointer` -> `get_DagBuilder_name_from_string`

### CODE REVIEW

File: dataflow/core/test/test_utils.py [hunk 1]
```diff
@@ -20,37 +20,32 @@ def test2(self) -> None:
         self.check_string(info)
 
 
-class Test_get_DagBuilder_name_from_string_pointer(hunitest.TestCase):
+class Test_get_DagBuilder_name_from_string(hunitest.TestCase):
     """
-    Test that `get_DagBuilder_name_from_string_pointer()` returns a correct DAG
-    builder name.
+    Test that `get_DagBuilder_name_from_string()` returns a correct DAG builder
```
Comment (rheenina) [inline dataflow/core/test/test_utils.py:25]: ...that the function returns ....

**Review by rheenina — APPROVED** [review]
1 nit and merge; feel free to merge yourself

**Comment by rheenina** [comment]
@PomazkinG FYI

**Review by gpsaggese — COMMENTED** [review]
LG

**Pull request merged**